modules/mono/editor/godotsharp_builds.cpp

```diff
@@ -49,9 +49,12 @@ void godot_icall_BuildInstance_ExitCallback(MonoString *p_solution, MonoString *
 #ifdef UNIX_ENABLED
 String _find_build_engine_on_unix(const String &p_name) {
 	String ret = path_which(p_name);
+	String ret_fallback = path_which(p_name + ".exe");
```

Comment:
I would move this after checking the result of `ret`, to avoid searching twice unnecessarily.